pkg/config/config.go

```diff
@@ -59,18 +59,31 @@ func getConfigurationFilePathFromEnvironment() string {
 // as the reference to the interactive.Oc instance, the reference to the test configuration, and the default network
 // IP address.
 type Container struct {
-	ContainerConfiguration  configsections.ContainerConfig
```

Comment:
ContainerConfiguration is used by:
https://github.com/test-network-function/test-network-function/pull/511
Why not keep it ?

Reply:
ok, I see this is an anonymous field, cool. 
http://golangtutorials.blogspot.com/2011/06/anonymous-fields-in-structs-like-object.html